client/my-sites/site-settings/controller.js

```diff
@@ -1,37 +1,44 @@
 /**
  * External Dependencies
  */
-var ReactDom = require( 'react-dom' ),
-	React = require( 'react' ),
-	page = require( 'page' );
+import ReactDom from 'react-dom';
+import React from 'react';
+import page from 'page';
 
 /**
  * Internal Dependencies
  */
-var sites = require( 'lib/sites-list' )(),
-	route = require( 'lib/route' ),
-	i18n = require( 'lib/mixins/i18n' ),
-	config = require( 'config' ),
-	analytics = require( 'lib/analytics' ),
-	titlecase = require( 'to-title-case' ),
-	SitePurchasesData = require( 'components/data/purchases/site-purchases' ),
-	SiteSettingsComponent = require( 'my-sites/site-settings/main' ),
-	DeleteSite = require( './delete-site' ),
-	StartOver = require( './start-over' ),
-	utils = require( 'lib/site/utils' ),
-	titleActions = require( 'lib/screen-title/actions' );
+import sitesFactory from 'lib/sites-list';
+import route from 'lib/route';
+import i18n from 'lib/mixins/i18n';
+import config from 'config';
+import analytics from 'lib/analytics';
+import titlecase from 'to-title-case';
+import SitePurchasesData from 'components/data/purchases/site-purchases';
+import { SiteSettingsComponent } from 'my-sites/site-settings/main';
+import DeleteSite from './delete-site';
+import StartOver from './start-over';
+import utils from 'lib/site/utils';
+import titleActions from 'lib/screen-title/actions';
+
+/**
+ * Module vars
+ */
+const sites = sitesFactory();
 
 function canDeleteSite( site ) {
-	return site.capabilities && site.capabilities.manage_options && ! site.jetpack && ! site.is_vip;
+	return site.capabilities &&
```

Comment:
I think compound conditionals like this are difficult to understand, even if they're broken apart onto different lines (which definitely is an improvement). But I think this would be easier to understand as:
```js
function canDeleteSite( site ) {
  if ( ! site.capabilities || ! site.capabilities.manage_options ) {
    return false;
  }
  if ( site.jetpack ) {
    return false;
  }
  if ( site.is_vip ) {
    return false;
  }
  return true;
}
```